.env.sample

```diff
@@ -22,3 +22,7 @@
 # The default cache expiration time in seconds if none is set
 # (default=60)
 #EXPIRATION_TIME_DEFAULT_SECONDS
+
+# Authorization token to use privileged endpoints.
+# The AUTH_TOKEN should always be set
```

Comment:
If `AUTH_TOKEN` is not setup, we will only get the error once an authorised route is triggered. We should consider using something like `onApplicationBootstrap()` lifecycle events to check if the setup is as expected.

No need to tackle this in this PR 👍

Reply:
Yeah, good point, I've just created a new issue to tackle this: https://github.com/5afe/safe-client-gateway-nest/issues/299